user-guide/early-access.md

```diff
@@ -6,17 +6,17 @@ Looking for early access to the Ambassador Edge Stack? [Click here to get starte
 
 From time to time, the Ambassador Edge Stack may ship early access releases to test major changes. **Early access releases are not supported for production use** but are intended to gain early feedback from our community before shipping a release.
 
-Early access releases will always have names that include the string "-ea" followed by a build number. For example, `0.50.0-ea1` is the first early access build of the Ambassador Edge Stack 0.50.0.
+Early access releases will always have names that include the string "-ea" followed by a build number.
 
 ## Early Access Status
 
-There are currently no early access releases available.
+The current early access build for the Ambassador Edge Stack is `1.0.0-ea1`.
```

Comment:
You should spell `1.0.0-ea1` as `$version$`, so it doesn't need replaced in multiple places.

Reply:
Fixed the incorrect numbering. But to your other point, isn't the point of this page showing users how to access an early release? $version$ is just the latest release.

Reply:
There are separate EA and GA `versions.yml` files; the EA one should have the EA version.  If that isn't currently the case, we should update the `versions.yml` file.

Reply:
fixed